src/components/Menu.css

```diff
@@ -26,7 +26,8 @@ ion-menu.md ion-list-header {
   padding-right: 18px;
 
   text-transform: uppercase;
-  letter-spacing: .1em;
+  letter-spacing: 0.1em;
+  font-size: min(0.875rem, 32px);
```

Comment:
The Angular conference app has a max font size of 36px: https://github.com/ionic-team/ionic-conference-app/pull/1805/files#diff-f17ac4f38c432797674ccd8854f9ce708922f6293f405ebc129414c13ec49c23R27

Any reason for the difference?

Reply:
Yes, but I didn't look into what it was. 36 worked there, so I tried to use it here too, but it was too large. I can change that one to 32 for consistency, if you'd like.

Reply:
Can you look into why there's a difference?

Reply:
Turns out they both need to be 32. I'm not sure what I changed on my end... or why 36 is working fine on your end 🤔